projects/README.md

```diff
@@ -2,16 +2,15 @@
+|[HDF-GROUP](http://www.hdfgroup.org 'The HDF Group')|[open](https://www.hdfgroup.org/ftp/HDF5/current/src/unpacked/COPYING 'HDF5 License')|HDF5 is a data model, library, and file format for storing and managing data|1.8.16|none|none|
+|[librdkafka](https://github.com/edenhill/librdkafka 'librdkafka on github')|[open](https://github.com/edenhill/librdkafka/blob/master/LICENSE '2-clause BSD license')|librdkafka is a C library implementation of the Apache Kafka protocol, containing both Producer and Consumer support|v0.9.5|[repo](https://github.com/distributePro/librdkafka 'distributePro fork of librdkafka repo on github')|[diff](https://github.com/distributePro/librdkafka/compare/edenhill:v0.9.5...xp-v0.9.5 'patch/diff')|
+|[NOVAS](http://aa.usno.navy.mil/software/novas/novas_info.php 'NOVAS')|[open](http://aa.usno.navy.mil/software/novas/novas_c/README.txt '(See Section IV. Using NOVAS in Your Applications)')|NOVAS is an integrated package of ANSI C functions for computing many commonly needed quantities in positional astronomy.|3.1|none|none|
 |[OpenH264](http://www.openh264.org/ 'OpenH264 website')|[open](http://http://www.openh264.org/LICENSE.txt 'Two-Clause BSD')|OpenH264 is a codec library which supports H.264 encoding and decoding. It is suitable for use in real time applications such as WebRTC.|1.4.0|[repo](https://github.com/distributepro/openh264 'forked openh264 repo on github')|[diff](https://github.com/distributepro/openh264/compare/cisco:v1.4.0...xp-v1.4.0 'patch/diff')|
 |[PostgreSQL](http://www.postgresql.org/ 'PostgreSQL')|[open](http://www.postgresql.org/about/licence 'PostgreSQL license')|PostgreSQL redistributable binaries|9.4.9|[repo](https://github.com/postgres/postgres 'Mirror of the official PostgreSQL GIT repository on github')|none|
-|[Qt](http://qt.io/ 'Qt - Home')|[LGPL](http://www.qt.io/qt-licensing-terms/ 'LGPL')|One Qt Code: Create Powerful Applications & Devices|5.8.0|[repo](http://code.qt.io/cgit/qt/qt5.git 'Qt5 main repo')|[diff](https://github.com/distributePro/ovsrpro/blob/master/patches/qt5.patch 'patch/diff')|
-|[Qwt](http://http://qwt.sourceforge.net/ 'Qwt - Qt Widgets for Technical Applications')|[LGPL](http://qwt.sourceforge.net/qwtlicense.html 'LGPL with exceptions')|The Qwt library contains GUI Components and utility classes which are primarily useful for programs with a technical background.|6.1.2|none|[diff](https://github.com/distributePro/ovsrpro/blob/master/patches/qwtconfig.pri.patch 'patch/diff')|
-|[ZeroMQ](https://zeromq.org/ 'ZeroMQ - Home")|[LGPL](https://github.com/zeromq/libzmq/blob/master/README.md/ 'LGPL v3')|ZeroMQ is a lightweight messaging kernel that extends standard socket interfaces with more powerful features.|4.2.2|[repo](https://github.com/zeromq/libzmq.git)|none|
```

Comment:
You could also fix this entry - change the `"` after Home to a `'` and it should fix the issue with it not displaying properly (I believe that will resolve issue #27)